src/app/features/StepDefinitionEditor/Services/MockParserService.js

```diff
@@ -49,10 +49,10 @@ var MockParserService = function MockParserService (
     }
 
     function parseUrl (mock, mockCallExpression) {
-        var url = _.last(mockCallExpression.callee.object.arguments).raw;
-        var urlRegex = new RegExp(url.replace(/^\//, '').replace(/\/$/, ''));
-        assert(urlRegex);
-        return urlRegex.source;
+        var rawUrl = _.last(mockCallExpression.callee.object.arguments).raw; //read raw url from json files

```

Comment:
Again, put comments on preceding line and add a space after the `//`. Change to `// The returned url should be the inner contents of the RegExp, with all escape sequences removed, so we strip out leading and trailing `/` characters, and any occurences of `//`.